server/src/authorizers/permissions.js

```diff
@@ -83,7 +112,10 @@ const rules = {
     channelCreate: isAuthenticated,
     userUpdate: isAuthenticated,
     seriesCreate: and(isAuthenticated, or(isChannelPublic, isAdmin)),
-    measurableCreate: and(isAuthenticated, or(isChannelPublic, or(isAdmin, isViewer))),
+    measurableCreate: and(
+      isAuthenticated,
+      or(isChannelPublic, or(isAdmin, isViewer)),
```

Comment:
Can we change "isChannelPublic" to "channelIsPublic", to go along with the others?

Reply:
Updated.